README.md

```diff
@@ -77,7 +77,7 @@ Your project APIs will be served at:
         -F "file=@www.tar.gz"
     ```
 
-    The website to publish is a `tar.gz` archive created using `tar -czvf www.tar.gz -C www .` and it may take up to half an hour for all URLs to become available.
+    The website to publish is a `tar.gz` archive created using `tar -czv --dereference --no-same-permissions -f www.tar.gz -C www .` and it may take up to half an hour for all URLs to become available.
```

Comment:
`--no-same-permissions` wont work for create, it won't works for extract.
`--dereference` shorthand is `-h` so `-czvhf` can be used

Also note this is default for "ordinary users" (non root)
```
       --no-same-permissions
              Apply the user's umask when extracting permissions from
              the archive (default for ordinary users).
```

Reply:
```suggestion
    The website to publish is a `tar.gz` archive created using `tar -czv --dereference  -f www.tar.gz -C www .` and it may take up to half an hour for all URLs to become available.
```

Reply:
```suggestion
    The website to publish is a `tar.gz` archive created using `tar -czhv -f www.tar.gz -C www .` and it may take up to half an hour for all URLs to become available.
```